config/simon/10002080-13X.xml

```diff
@@ -0,0 +1,354 @@
+      <Item label="95 minutes down time" value="222"/>
+      <Item label="96 minutes down time" value="223"/>
+      <Item label="97 minutes down time" value="224"/>
+      <Item label="98 minutes down time" value="225"/>
+      <Item label="99 minutes down time" value="226"/>
+      <Item label="100 minutes down time" value="227"/>
+      <Item label="101 minutes down time" value="228"/>
+      <Item label="102 minutes down time" value="229"/>
+      <Item label="103 minutes down time" value="230"/>
+      <Item label="104 minutes down time" value="231"/>
+      <Item label="105 minutes down time" value="232"/>
+      <Item label="106 minutes down time" value="233"/>
+      <Item label="107 minutes down time" value="234"/>
+      <Item label="108 minutes down time" value="235"/>
+      <Item label="109 minutes down time" value="236"/>
+      <Item label="110 minutes down time" value="237"/>
+      <Item label="111 minutes down time" value="238"/>
+      <Item label="112 minutes down time" value="239"/>
+      <Item label="113 minutes down time" value="240"/>
+      <Item label="114 minutes down time" value="241"/>
+      <Item label="115 minutes down time" value="242"/>
+      <Item label="116 minutes down time" value="243"/>
+      <Item label="117 minutes down time" value="244"/>
+      <Item label="118 minutes down time" value="245"/>
+      <Item label="119 minutes down time" value="246"/>
+      <Item label="120 minutes down time" value="247"/>
+      <Item label="121 minutes down time" value="248"/>
+      <Item label="122 minutes down time" value="249"/>
+      <Item label="123 minutes down time" value="250"/>
+      <Item label="124 minutes down time" value="251"/>
+      <Item label="125 minutes down time" value="252"/>
+      <Item label="126 minutes down time" value="253"/>
+      <Item label="127 minutes down time" value="254"/>
+    </Value>
+
+    <Value genre="config" index="12" label="Behavior of LED in Repose" max="255" min="0" size="1" type="list" value="0">
+      <Help>This parameter allows to choose behaviour of LED in repose</Help>
+      <Item label="LED OFF" value="0"/>
+      <Item label="LED ON at 20% of maximum level" value="255"/>
+    </Value>
+
+    <Value genre="config" index="13" label="Lock Input" max="255" min="0" size="1" type="list" value="0">
+      <Help>This parameter locks the direct control of load.</Help>
+      <Item label="Unlock the direct control of load" value="0"/>
+      <Item label="Lock the direct control of load" value="255"/>
+    </Value>
+
+    <Value genre="config" index="15" label="Reset Default" max="39015" min="39015" size="2" type="list" value="0" write_only="true">
+      <Help>Parameters, Groups, and Z-Wave status are restored to default (Only writable).</Help>
+      <Item label="Device performs a factory default Reset" value="39015"/>
+    </Value>
+
+    <Value genre="config" index="20" label="Identify (Write Only)" max="255" min="255" size="1" type="list" value="0">
```

Comment:
bool?

Reply:
I would keep this definition since Simon's 10002041_13X and 10002034_13X (from same family) have the same specification for this values.